lib/gamnit/README.md

```diff
@@ -0,0 +1,46 @@
+Portable game and multimedia framework for Nit
+
+_gamnit_ is a modular framework to create portable 2D or 3D apps in Nit.
+It is based on the portability framework _app.nit_ and the OpenGL ES 2.0 standard.
+
+# System configuration
+
+To compile the _gamnit_ apps packaged with the Nit repositoy on GNU/Linux you need to install the dev version of a few libraries and some tools.
+Under Debian 8.2, this command should install everything needed: `apt-get install libgles2-mesa-dev libsdl1.2-dev libsdl-image1.2-dev libsdl-ttf2.0-dev inkscape mpg123`
```

Comment:
you should put the command on an indented codeblock to simplify the indentification and the copy-pasting